src/ImageSharp.Drawing/Processing/Processors/Drawing/FillPathProcessor.cs

```diff
@@ -32,9 +36,9 @@ public FillPathProcessor(DrawingOptions options, IBrush brush, IPath shape)
         public IBrush Brush { get; }
 
         /// <summary>
-        /// Gets the region that this processor applies to.
+        /// Gets the logic path that this processor applies to.
         /// </summary>
-        public IPath Shape { get; }
+        public IPath Path { get; }
```

Comment:
I think "region" is a better name when it's about filling:

```suggestion
        public IPath Region { get; }
```